client/src/components/Nav.tsx

```
@@ -41,14 +40,13 @@ function Nav() {
-        <div
+        <button
           className="block sm:hidden cursor-pointer"
           onClick={toggleMobileMenu}
-          role="button"
           aria-label="Toggle Mobile Menu"
           tabIndex={0}
           onKeyDown={(e) => {
-            if (e.key === "Enter" || e.key === " ") toggleMobileMenu();
+            if (e.key === 'Enter' || e.key === ' ') toggleMobileMenu();
           }}
         >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)